Documentation/data-collection.md

```diff
@@ -417,6 +417,48 @@ data:
     # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="console_url"}'
     #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_requests_total:sum gives the total number of login requests initiated from the web console.
+    #
+    - '{__name__="cluster:console_auth_login_requests_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_successes_total:sum gives the total number of successful logins initiated from the web console.
+    # Role is kubeadmin, cluster-admin or developer based on a permission check if ther user can list all namespaces.
+    #
+    - '{__name__="cluster:console_auth_login_successes_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_failures_total:sum gives the total number of login failures initiated from the web console.
+    # This might include canceled OAuth logins depenending on the user OAuth provider/configuration.
```

Comment:
```suggestion
    # This might include canceled OAuth logins depenending on the user OAuth provider/configuration.
    # Labels:
    # * `reason`, one of ...
```